CRM/Case/Form/ActivityToCase.php

```diff
@@ -48,6 +48,15 @@ public function preProcess() {
     $this->_currentCaseId = CRM_Utils_Request::retrieve('caseId', 'Positive');
     $this->assign('currentCaseId', $this->_currentCaseId);
     $this->assign('buildCaseActivityForm', TRUE);
+
+    switch (CRM_Utils_Request::retrieve('focaction', 'String')) {
+      case 'move':
+        CRM_Utils_System::setTitle(ts('Move to Case'));
+        break;
```

Comment:
@mattwire Case breaking statements must be followed by a single blank line

Reply:
Thankyou, fixed.